index.html

```diff
@@ -0,0 +1,455 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+    <script type="text/javaScript" src="js/main.js"></script>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" onclick="showNav()">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+      <nav id="mobile-nav">
+        <i class="fas fa-times" id="close-btn" onClick="hideNav()"></i>
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li><a class="nav-link" href="#contact">Contact</a></li>
```

Comment:
- [ ] When clicking on the links your menu does not disappear, Please add this requirement :wink: 
![dis](https://user-images.githubusercontent.com/75126481/135994845-cf98ee43-6b66-4fd1-ae89-ca519bb3835f.png)

Reply:
Done